do/common.go

```diff
@@ -0,0 +1,53 @@
+package do
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/digitalocean/godo"
+	"github.com/digitalocean/godo/context"
+)
+
+const apiPerPage = 100
+
+func allDropletList(ctx context.Context, client *godo.Client) ([]godo.Droplet, error) {
+	// create a list to hold our droplets
+	list := []godo.Droplet{}
+
+	// create options. initially, these will be blank
+	opt := &godo.ListOptions{PerPage: apiPerPage}
+	for {
+		droplets, resp, err := client.Droplets.List(ctx, opt)
+		if err != nil {
+			return nil, err
+		}
+
+		if resp == nil {
+			return nil, fmt.Errorf("DO API returned no response ")
```

Comment:
Since we won't probably have a trace at logs of where we are, we could replace this message for something like
```
droplets list request returned no response
```